### DISCUSSION TITLE
Evolution 1 Wiring help

### DISCUSSION THREAD

**Discussion by MarkKananen** [jgyates/genmon#1153] [Questions] [answered]
### Checklist

- [X] I made sure that there are *no existing issues or discussions* - [open issues](https://github.com/jgyates/genmon/issues),[closed issues](https://github.com/jgyates/genmon/issues?q=is%3Aissue+is%3Aclosed),[open discussion](https://github.com/jgyates/genmon/discussions), [closed discussion](https://github.com/jgyates/genmon/discussions/categories/questions?discussions_q=is%3Aclosed) - which may answer my question or that I could contribute my information to.
- [X] I have read the [Known Issues Wiki Page](https://github.com/jgyates/genmon/wiki/Appendix-D-Known-Issues) and my question / issue is not listed.
- [X] I have provided the version of the software and the operating system in my post.
- [X] If I have an issue with genmon not communicating with the controller, I have read and followed the [serial troubleshooting guide](https://github.com/jgyates/genmon/wiki/3.6---Serial-Troubleshooting).

### Version

latest

### Have you submitted your log files from the About page? Note that outbound email must be working for logs to be submitted.

I have submitted logs

### What is/are your question(s)?

So I finally have genmon configured and running on a pie zero.  I went to plug it into the generac evolution 1 controller and the molex does not match up.  I bought the kit from pintsize and specified evolution 1 controller.  An guidance appreciated... 

### Additional information

![back1](https://github.com/user-attachments/assets/1e3ee14b-bdac-4b83-ad6c-379e0b1deb8b)
![back2](https://github.com/user-attachments/assets/4ad20191-2386-4d6b-81bc-d19edcdaa805)
![e1](https://github.com/user-attachments/assets/0d33b388-f3fb-442d-8425-8299a256a6c8)
![pi](https://github.com/user-attachments/assets/d83ca946-8e48-4c29-875a-11b03cfb57ca)

**Comment by MarkKananen** [comment]
I tied unplugging accessory port 1, and thee generator would not auto start.  It had an RPM error.

**Reply by skipfire** [reply]
~~RPM sense loss is typically a battery or a starter error.  You should be able to clear the error from the screen and try to start it.~~

**Reply by skipfire** [reply, marked as answer]
Actually I think it's a different thing.  Those stickers are not labels, they are covers.  Accessory Port 1 is covered by the sticker not above it.

**Reply by MarkKananen** [reply]
it is raining tonight, I'll go take it apart again tomorrow.  I sure hope that is the case!  Thanks

**Comment by MarkKananen** [comment]
Ok- I finally got back to this.  Sure enough it was a sticker.  I plugged it in and everything seems to work fine.  Wifi signal in the generator housing is pretty weak, but holding.  When I start the generator from the maintenance page, it starts.  But then it shuts o and goes into some sort of restart loop.  It will run or like 10 seconds, shuts down waits 10 seconds.  Rinse and repeat.  Any ideas?

**Reply by jgyates** [reply]
How are you starting on the maintenance page? Start, Start and Transfer? 

I would look at the Run log. The controller will log the reason for starts and stops if they are initiated by external commands (like genmon does), via a button press or due to an outage. If it is not logged then I would look at the LCD screen on the controller when this is occurring as this may give some insight.

**Reply by MarkKananen** [reply]
I tied again.  I hit the start button (no transfer).   It starts and runs for 20-30 seconds while reading "RUNNING Warming up".  It then shuts down and says "CRANKING - Pause for NN Sec"  and counts down.  When it reaches 0 it starts again.  After the second start I tried a Stop from the Maintenance page, it seemed to run for the 20-30 seconds but did not restart.

**Reply by jgyates** [reply]
Are you running two wire mode or do you have a typical automatic transfer switch?

On the Outage page, what is the value of Start Delay and Warm Up delay? The defaults are 5

What happens if you manually start (via the button on the controller)?

I assume that you only have one Start (something like Remote Start) in the Run log? The second start is not in the log, correct?

**Reply by MarkKananen** [reply]
I have a transfer switch.  I do not know if it is two wire?
The outage page only has a 15s startup delay and no warmup delay.
When I started manual, I had issues.  First I got a code 1900 (voltage), tied again and got a 1600 (RPM). So I disconnected the molex and tied again.  Same issues.  I fiddled around with some of the other wires and found one that seemed loose.  I firmed it up and things seem to be working.   Start/Stop seem to work OK.   I'll send the log anyway, but thanks or the help.  You have been able to steer me around really well!

**Reply by jgyates** [reply]
I am glad it is working for you.
